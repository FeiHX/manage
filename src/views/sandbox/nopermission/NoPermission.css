#app{
    padding: 1rem;
    display: flex;
    height: 100%;
    justify-content: center; 
    align-items: center;
    color: red;
    font-size: 10rem;
    flex-direction: column;
    text-shadow: 0px 0px 10px ;
    .txt {
        font-size: 3rem;
    }
}
@keyframes blink {
    0%   {opacity: 0}
    49%  {opacity: 0}
    50%  {opacity: 1}
    100% {opacity: 1}
}
.blink {
    animation-name: blink;
    animation-duration: 1s;
    animation-iteration-count: infinite;
}
