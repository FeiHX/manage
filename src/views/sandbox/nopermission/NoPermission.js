import React, { useEffect, useState } from "react";
import "./NoPermission.css";

export default function NoPermission() {
  return (
    <div id="app">
      <div>403</div>
      <div className="txt">
        Forbidden
        <span className="blink">_</span>
      </div>
    </div>
  );
}
