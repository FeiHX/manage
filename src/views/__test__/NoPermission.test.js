import React from 'react';
import { render } from '@testing-library/react';
import NoPermission from '../sandbox/nopermission/NoPermission';
describe('NoPermission Test', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<NoPermission />);
    expect(asFragment()).toMatchSnapshot();
  });
});
