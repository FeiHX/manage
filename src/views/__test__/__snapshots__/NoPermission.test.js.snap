// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPermission Test renders correctly 1`] = `
<DocumentFragment>
  <div
    id="app"
  >
    <div>
      403
    </div>
    <div
      class="txt"
    >
      Forbidden
      <span
        class="blink"
      >
        _
      </span>
    </div>
  </div>
</DocumentFragment>
`;

exports[`NoPermission renders correctly 1`] = `
<DocumentFragment>
  <div
    id="app"
  >
    <div>
      403
    </div>
    <div
      class="txt"
    >
      Forbidden
      <span
        class="blink"
      >
        _
      </span>
    </div>
  </div>
</DocumentFragment>
`;
